### PR TITLE
138 create a mmsa epn functionality notebook

### PR DESCRIPTION
**Pull request by longshuicy**

### CODE REVIEW

File: manual_jb/content/analyses/epn_functionality.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    epn_func.load_remote_input_dataset("epf_sample_failure_state", epf_sample_failure_state_id)
+
+    epn_func.set_parameter("result_name", result_name)
+    epn_func.set_parameter("gate_station_node_list", [1, 2, 3, 4, 5, 6, 7, 8, 9])
+
+    # Run Analysis
+    epn_func.run_analysis()
+```
+
+full analysis: [epn_functionality.ipynb](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/epn_functionality.ipynb)
```
Comment (navarroc) [inline manual_jb/content/analyses/epn_functionality.md:68]: Can you change master to main?

File: manual_jb/content/analyses/epn_functionality.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+**Related publications**
+
+* Sharma, N., Tabandeh, A., & Gardoni, P. (2019). Regional resilience analysis: A multi-scale approach to model the 
+  recovery of interdependent infrastructure. In P. Gardoni (Ed.), *Handbook of sustainable and resilient 
+  infrastructure* (pp. 521–544). New York, NY: Routledge.
+* Sharma, N., Tabandeh, A., & Gardoni, P. (2020). Regional resilience analysis: A multi-scale approach to optimize 
+  the resilience of interdependent infrastructure. *Computer‐Aided Civil and Infrastructure Engineering*, **35(12)**, 
+  1315-1330.
+* harma, N., & Gardoni, P. (2022). Mathematical modeling of interdependent infrastructure: An object-oriented
```
Comment (navarroc) [inline manual_jb/content/analyses/epn_functionality.md:25]: Missing S in Sharma

**Comment by navarroc** [comment]
Besides the typo, this looks good. Notebook runs without issue. Can you fix the conflict with analyses.md?

**Review by navarroc — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
```
Comment (navarroc) [inline CHANGELOG.md:8]: Can you remove the extra Added section?

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**